lib/appsignal/config.rb

```diff
@@ -90,13 +91,18 @@ def []=(key, value)
 
     def log_file_path
       path = config_hash[:log_path] || root_path
-      if path && File.writable?(path)
-        File.join(File.realpath(path), 'appsignal.log')
+      return File.join(path, 'appsignal.log') if path && File.writable?(path)
```

Comment:
The `File.realpath` call is essential because it allows us to break out Capistrano style release directories to shared directories. Without this an agent that survives enough deploys to live through a cleanup dies.